include/seqan3/alphabet/nucleotide/sam_dna16.hpp

```diff
@@ -0,0 +1,132 @@
+// -----------------------------------------------------------------------------------------------------
+// Copyright (c) 2006-2019, Knut Reinert & Freie Universität Berlin
+// Copyright (c) 2016-2019, Knut Reinert & MPI für molekulare Genetik
+// This file may be used, modified and/or redistributed under the terms of the 3-clause BSD-License
+// shipped with this file and also available at: https://github.com/seqan/seqan3/blob/master/LICENSE.md
+// -----------------------------------------------------------------------------------------------------
+
+/*!\file
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
+ * \brief Contains seqan3::sam_eq and seqan3::sam_dna16.
+ */
+
+#pragma once
+
+#include <cassert>
+
+#include <seqan3/alphabet/composite/alphabet_variant.hpp>
+#include <seqan3/alphabet/detail/alphabet_base.hpp>
+#include <seqan3/alphabet/nucleotide/dna15.hpp>
+
+namespace seqan3
+{
+
+/*!\brief The alphabet of an equal character '='.
+ * \ingroup nucleotide
+ * \implements seqan3::Alphabet
+ * \implements seqan3::detail::ConstexprAlphabet
+ * \implements seqan3::TriviallyCopyable
+ * \implements seqan3::StandardLayout
+ *
+ * The alphabet always has the same value ('=') which represents that the base
+ * at this position is equal to the aligned reference base.
+ */
+
+class sam_eq : public alphabet_base<sam_eq, 1, char>
+{
+private:
+    //!\brief The base class.
+    using base_t = alphabet_base<sam_eq, 1, char>;
+
+    //!\brief Befriend seqan3::alphabet_base.
+    friend base_t;
+
+    //!\brief The character that will be printed.
+    static constexpr char char_value = '=';
+
+public:
+    /*!\name Constructors, destructor and assignment
+     * \{
+     */
+    constexpr sam_eq() noexcept : base_t{} {}            //!< Defaulted.
+    constexpr sam_eq(sam_eq const &) = default;             //!< Defaulted.
+    constexpr sam_eq(sam_eq &&) = default;                  //!< Defaulted.
+    constexpr sam_eq & operator=(sam_eq const &) = default; //!< Defaulted.
+    constexpr sam_eq & operator=(sam_eq &&) = default;      //!< Defaulted.
```

Comment:
```suggestion
    constexpr sam_eq & operator=(sam_eq &&)            noexcept = default;  //!< Defaulted.
```

Alsign the last three words. Also acutally default the default constructor